### DISCUSSION TITLE
Last few days my ESLint is not working correctly - freezing/long delay after detecting or fixing problem - both VS Code and Webstorm

### DISCUSSION THREAD

**Discussion by BaronFO** [eslint/eslint#17326] [Q&A]
Hello guys, I am really desperate. I couldnt do anything in work these last few days because of problems with ESLint. Everytime I make a ESLint mistake it takes ages to underline it and IDE would take gigabytes of RAM while communicating with server. I went through all possible fixes these days - increased memory in settings, reinstalled everything, tried various versions from all softwares, recloned repos, tried different config etc. My problem occurs with our biggest project (smaller projects work fine however I would say little bit slower than usual). It takes cca 20 seconds for error to get highlighted, it takes same time when I click on fix. I work in this project for 2 years and never had this problem before, jsut lately. I went through all possible discussions and troubleshooting, spent around 16 hours fixing it and without result. 

Is anybody experiencing this issue as well? Can it be caused by something outside my computer?

Webstorm takes more than 6gb of RAM while "linting"... it doesnt matter wheter it is run through console or UI

**Comment by Rec0iL99** [comment]
Hi @BaronFO, unfortunately, we don't maintain the ESLint extensions for VSCode and Webstorm. You can open an issue there and they should be able to help you.

**Comment by BaronFO** [comment]
Understood, thanks :)
